evap/settings.py

```diff
@@ -418,6 +419,10 @@
     # give random char field values a reasonable length
     BAKER_CUSTOM_FIELDS_GEN = {"django.db.models.CharField": lambda: random_gen.gen_string(20)}
 
+    # csrf tokens are added in JS and do not work in the tests at all. The django test runner
+    # does disable CSRF protection by default, but the WebTest app does not. So we disable it here.
+    MIDDLEWARE = [m for m in MIDDLEWARE if m != "django.middleware.csrf.CsrfViewMiddleware"]
```

Comment:
Why not `MIDDLEWARE.remove("django.middleware.csrf.CsrfViewMiddleware")`? This would give us an error if this was somehow not in there already

Reply:
because I didn't know that exists :)